pkg/virtual/initializingworkspaces/doc.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package initializingworkspaces and its sub-packages provide the Initializing Workspace Virtual Workspace.
+//
+// It allows for one basic functions:
+// - cross-cluster LIST + WATCH of Workspaces which:
+//   - are in the Initializing phase
+//   - request initialization by a specific controller
+//
+// That is, a request for
+// GET /services/initializingworkspaces/clusters/*/apis/tenancy.kcp.io/v1alpha1/clusterworkspaces
```

Comment:
Is this missing `<initializer-name>` in between `initializingworkspaces` and `clusters`?